### PR TITLE
fix: @W-11989149@ revert file naming and module: ... entry changes to restore coverage collection

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
I _think_ this should be fine for library consumers, but @jdalton can you confirm?

**Comment by rwaldron** [comment]
I'm baffled by this failure. It doesn't reproduce locally 😖

File: packages/near-membrane-dom/package.json [hunk 1]
```diff
@@ -21,13 +21,13 @@
...
-            "import": "./dist/index.mjs",
-            "require": "./dist/index.js"
+            "import": "./dist/index.js",
+            "require": "./dist/index.cjs.js"
         },
         "./*.js": {
-            "import": "./dist/*.mjs",
-            "require": "./dist/*.js",
-            "default": "./dist/*.js"
+            "import": "./dist/*.js",
```
Comment (jdalton) [inline packages/near-membrane-dom/package.json:28]: ☝️ the issue here is that *.js matches *.cjs.js

Comment (jdalton) [reply packages/near-membrane-dom/package.json:28]: What about `*.cjs.js` and `*.mjs.js`

Comment (rwaldron) [reply packages/near-membrane-dom/package.json:28]: Let's try it!

**Comment by jdalton** [comment]
Look at the eslint.cjs file configs patterns

File: packages/near-membrane-base/src/__tests__/connector.spec.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { createBlueConnector, createRedConnector } from '../../dist/index';
+import { createBlueConnector, createRedConnector } from '../../dist/index.mjs';
```
Comment (jdalton) [inline packages/near-membrane-base/src/__tests__/connector.spec.ts:1]: `index.mjs` won't exist. It's now `index.mjs.js`

**Review by jdalton — APPROVED** [review]

**Pull request merged**